fetch/cosmwasm_contract/src/access_control.rs

```diff
@@ -2,11 +2,15 @@ use cosmwasm_std::{HumanAddr, ReadonlyStorage, StdError, StdResult, Storage};
 use cosmwasm_storage::{PrefixedStorage, ReadonlyPrefixedStorage};
 use std::str::{from_utf8, FromStr};
 
+use crate::error::{ERR_ACCESS_CONTROL_ALREADY_HAVE_ROLE, ERR_ACCESS_CONTROL_DONT_HAVE_ROLE};
+
 pub static ACCESS_CONTROL_KEY: &[u8] = b"access_control";
 
 pub const OWNER_KEY: &[u8] = b"OWNER_KEY";
```

Comment:
I believe this should be removed - it is not used for anything functional in the contract anymore (besides setting it during contract initialisation).
I mean this is not critical, nothing will happen if it stays in, just that it will be confusing later on for people when they will be reading contract code.

Reply:
it is used by the module where it is declared. It is used by `ac_set_owner`, `ac_get_owner`.
I understand that now that owner is no more a special case we can ditch it, but didn't want to do it at critical time.

Reply:
removed it